src/content/vintage/advanced/access-management/configure-dex-in-your-cluster/index.md

```diff
@@ -401,7 +401,7 @@ kubernetes:
     M...=
     -----END CERTIFICATE-----
   api:
-    address: https://api.test.example.io
+    address: https://api.test.example.io:6443
```

Comment:
With CAPA, we default to Kubernetes API port 443 by now (changed from 6443)

Reply:
Thanks, added note to use 443 for CAPA.